Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -99,6 +99,54 @@ open class ChatMessageContentView: _View, ThemeProvider, UITextViewDelegate {
     /// for the content will taken from the provided `width`.
     open var messageAuthorAvatarSize: CGSize { .init(width: 32, height: 32) }
 
+    /// The font used when rendering emojis as "Jumbomoji".
+    open var jumbomojiMessageFont: UIFont {
+        appearance.fonts.emoji
+    }
+
+    /// The font used when rendering system messages.
+    open var systemMessageFont: UIFont {
+        appearance.fonts.caption1.bold
+    }
+
+    /// The default font when rendering the message text.
+    ///
+    /// **Note:** Because of an issue which we don't know yet the root cause, if you want
```

Comment:
Don't we know that it is because of UIKit's internal caching system?

Reply:
We don't 100% know yet. Because for the TimestampLabel this works fine :/